src/ReadingBuddy.java

```diff
@@ -804,40 +853,49 @@ public static void chooseFromAllBooks(){
             System.out.print("\nEnter the corresponding number to select a book to find a book buddy (make a Book Buddy" +
              "Wanted post or answer someone else's), or 0 to return to the menu:");
             menuSelection = validateMenuSelection(0, allBooks.size());
+            //TESTING
```

Comment:
anything with //TESTING is stuff I forgot to remove before debugging (I will clean that up)